src/components/appointments/AppointmentForm.js

```diff
@@ -175,31 +175,20 @@ const AppointmentForm = ({form}) => {
 
   // Route the correct page based on URL
   const router = (
-    <Switch>
-      <Route path="/appointment/overzicht">
-        <ErrorBoundary useCard>
-          <AppointmentSummary form={form} onConfirm={onSubmitForm} />
-        </ErrorBoundary>
-      </Route>
-
-      <Route path="/appointment/bevestiging">
-        <ErrorBoundary useCard>
-          <AppointmentConfirmation />
-        </ErrorBoundary>
-      </Route>
-
-      <Route path="/">
-        <ErrorBoundary useCard>
-          <AppointmentStep form={form} onSubmit={onAppointmentSubmit} />
-        </ErrorBoundary>
-      </Route>
-    </Switch>
+    <Routes>
+      <Route
+        path="overzicht"
+        element={<AppointmentSummary form={form} onConfirm={onSubmitForm} />}
+      />
+      <Route path="bevestiging" element={<AppointmentConfirmation />} />
+      <Route path="" element={<AppointmentStep form={form} onSubmit={onAppointmentSubmit} />} />
+    </Routes>
   );
 
   const FormDisplayComponent = config?.displayComponents?.form ?? FormDisplay;
   return (
     <FormDisplayComponent
-      router={router}
+      router={<ErrorBoundary useCard>{router}</ErrorBoundary>}
```

Comment:
no particular reason, I just moved things around without thinking to much. This `AppointmentForm` code is not used in prod right now anyway, and will change in the next few weeks so I'll tackle that as part of appointments work.

In other situations sometimes the error boundary should not be wrapped in a card, and sometimes it does, depending on the route, so for those you'd specify it per route instead of wrapping the entire component.

That said - there's also an argument to put your error boundary only around your own code/components and not around the third party library, as it could mask bugs in the router library.